tests/features/rhpam/businesscentral/rhpam-businesscentral.feature

```diff
@@ -9,7 +9,7 @@ Feature: RHPAM Business Central configuration tests
   Scenario: Check for product and version environment variables
     When container is ready
     Then run sh -c 'echo $JBOSS_PRODUCT' in container and check its output for rhpam-businesscentral
-     And run sh -c 'echo $RHPAM_BUSINESS_CENTRAL_VERSION' in container and check its output for 7.8.0
+     And run sh -c 'echo $RHPAM_BUSINESS_CENTRAL_VERSION' in container and check its output for 7.9.0
```

Comment:
In other feature scripts in step is used short form of version like '7.9'. It would be good to follow same format style also here.
```suggestion
     And run sh -c 'echo $RHPAM_BUSINESS_CENTRAL_VERSION' in container and check its output for 7.9
```